apps/tng-radio-hub/src/app/app.tsx

```diff
@@ -1,10 +1,22 @@
+import { Route, Routes } from 'react-router-dom';
+import About from '../Components/pages/about/about';
+import Contact from '../Components/pages/contact/contact';
+import Funders from '../Components/pages/funders/funders';
+import Home from '../Components/pages/home/home';
+import Links from '../Components/pages/links/links';
+import Stream from '../Components/pages/stream/stream';
 
 
```

Comment:
May need memory router in app.spec.tsx (imported from 'react-router-dom',
`
const { baseElement } = render(<MemoryRouter><App /></MemoryRouter>;
`